R/pkg/R/context.R

```diff
@@ -330,7 +330,13 @@ spark.addFile <- function(path, recursive = FALSE) {
 #'}
 #' @note spark.getSparkFilesRootDirectory since 2.1.0
 spark.getSparkFilesRootDirectory <- function() {
-  callJStatic("org.apache.spark.SparkFiles", "getRootDirectory")
+  if (Sys.getenv("IS_RUNNING_ON_WORKER") == "") {
```

Comment:
would likely better to prefix this with SPARKR or similar - to be consistent with other env variables and also hope to avoid conflict